src/dags/gob.py

```diff
@@ -89,12 +89,16 @@ def create_gob_dag(is_first: bool, gob_dataset_id: str, gob_table_id: str) -> DA
         )
 
         def _create_dataset_info(dataset_id: str, table_id: str) -> DatasetInfo:
-            dataset = schema_def_from_url(SCHEMA_URL, dataset_id)
+            dataset = schema_def_from_url(SCHEMA_URL, dataset_id, prefetch_related=True)
```

Comment:
Just a note: In de toekomst moeten we dit wijzingen wanneer we versioning gaan toepassen op schema's omdat de locaties van deze schema's wijzingen (subfolders met versienumbers en daarin schema file)

Reply:
The future is here given your recent PR https://github.com/Amsterdam/dataservices-airflow/pull/253 @cvriel